### PR TITLE
DCC-5118

### PR DESCRIPTION
**Pull request by grant-guo**
Assign ES document index tasks and tar creation tasks to different executor pools

### CODE REVIEW

File: dcc-release-job/dcc-release-job-index/src/main/java/org/icgc/dcc/release/job/index/core/IndexJob.java [hunk 1]
```diff
@@ -80,40 +83,49 @@ public void execute(JobContext jobContext) {
...
-    val bigFilesTask = allTasks.stream()
-        .filter(task -> task instanceof IndexBigFilesTask)
-        .findFirst();
-    if (bigFilesTask.isPresent()) {
-      log.info("Indexing big files...");
-      jobContext.execute(bigFilesTask.get());
-    }
+    jobContext.execute(indexTaskExecutor, indexTasks);
+
+//    val noBigFilesTasks = allTasks.stream()
```
Comment (andricDu) [inline dcc-release-job/dcc-release-job-index/src/main/java/org/icgc/dcc/release/job/index/core/IndexJob.java:115]: What happened to the big files task?

**Review by andricDu — CHANGES_REQUESTED** [review]

**Comment by andricDu** [comment]
We are going to test a full document + index and then merge if successful.

**Comment by grant-guo** [comment]
ok to test

**Comment by grant-guo** [comment]
ok to test

**Review by andricDu — APPROVED** [review]

**Pull request merged**